core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -538,7 +544,7 @@ public void testNamespaceWithDot() {
   public void testBasicCreateTable() {
     C catalog = catalog();
 
-    TableIdentifier ident = TableIdentifier.of("ns", "table");
+    TableIdentifier ident = TableIdentifier.of("ns", "tbl");
```

Comment:
if we're introducing `TBL` then we should replace the local `ident` with `TBL` here and in all other test methods

Reply:
Sounds good I did.